src/components/pages/Leaderboard/RenderLeaderboard.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Layout, Menu } from 'antd';
+import styled from 'styled-components';
+import { LayoutContainer } from '../../common/';
+import NavBar from '../../common/NavBar';
+
+const StyledList = styled.div`
```

Comment:
I'd love to see these Styled components in their own file and then imported into the component so we can reuse them elsewhere in the app. Also if these Styled components could be in their own StyledComponents folder possibly in the styles folder so we help keep our styles in one place.

Reply:
Created a new file named Leaderboard.css. Ended up not needing styled-components at all.